drivers/hmis/app/graphql/types/hmis_schema/enrollment.rb

```diff
@@ -59,13 +59,13 @@ def exit
     def household
       return nil unless object.household_id.present?
 
-      Hmis::Hud::Enrollment.where(household_id: object.household_id).preload(:client)
+      Hmis::Hud::Enrollment.where(household_id: object.household_id, data_source_id: object.data_source_id).preload(:client)
```

Comment:
Do you want to transition these to viewable_by?

Reply:
Yeah. I thought this was kind of different since it's in a resolver, but it's not. viewable_by still makes sense for consistency